flume-ng-sinks/flume-hive-sink/metastore_db/README_DO_NOT_TOUCH_FILES.txt

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Hi @rgoers , thanks for the pull request
I am wondering how this metastore_db folder is necessary to upgrade the log4j version.

Reply:
This is some leftover from a failed test run. Could you please remove it so we could review the real changes?

Reply:
I didn't modify those files so I'm not sure how they got in the commit. Out of curiosity, is there a reason they aren't copied to the target directory if the tests are going to modify them? Maven convention is that only items in the target directory should be modified by the build so I don't have my gitignore set up to skip these files.

Reply:
I will revert those files as soon as I can.